SI/Sem.02/Pract.02/Task9.cpp

```diff
@@ -0,0 +1,37 @@
+﻿// Task9.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int firstNum, secondNum, thirdNum;
+	cout << "Enter three numbers: ";
+	cin >> firstNum >> secondNum >> thirdNum;
+	if (firstNum < secondNum && firstNum < thirdNum && secondNum < thirdNum)
```

Comment:
This solution doesn't work for the input 5 6 5 and there is a much simpler solution with just reordering(swapping) the numbers with three ifs